sailor/sap_iot/wrappers.py

```diff
@@ -235,17 +237,25 @@ def plot(self, start=None, end=None, indicator_set=None, equipment_set=None):
             aggregation_interval = '1h'
         else:
             aggregation_interval = '12h'
-        groupers = [pd.Grouper(key=time_column, freq=aggregation_interval), *self.get_key_columns()]
```

Comment:
I just noticed that line 231 to 239 should not be used if we have an AggregatedIndicator. We should be able to just get the aggregation interval from it.

Reply:
I don't think it's necessarily the same aggregation interval actually.
The interval here is chosen such that the graph looks good, nothing else. If the `AggregateIndicator` has a 2-min interval, and you want to plot data for say a month here you still want the plotting function to aggregate to a longer interval. [30 days/month * 24 hours/day * 30 (2-min-aggregates/hour) = 21600 points, bit too much for the resolution of my screen :)]

Reply:
Ah sure. Thanks for clearing that up